### PR TITLE
Justinemolinos feature2

### PR DESCRIPTION
**Pull request by justinemolinos**
#1 
#2

### CODE REVIEW

File: doc/authors/justinemolinos.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+Nayti Nayti Nayti!
```
Comment (big-samantha) [inline doc/authors/justinemolinos.txt:2]: Why did you change this line. @datafry any thoughts?

**Review by big-samantha — APPROVED** [review]
This looks perfect. Thanks for reading the objective.

File: doc/authors/justinemolinos.txt (hunk 1 above)
Comment (datafry) [reply doc/authors/justinemolinos.txt:2]: I wuz here
